### ISSUE TITLE
Provide visual cue to distinguish quickly between program log entry and spin

### ISSUE THREAD

**Issue by RocketMan** [RocketMan/zookeeper#97]
The spins and log entries run together; it is difficult to pick them out easily.  For example,

https://zookeeper.stanford.edu:8443/?action=viewDate&seq=selList&playlist=39937&session=

Let's try something like setting it off in italics.  If that does not work, let's try other visual options.

**Comment by RocketMan** [comment]
Trying another visual option:  Updated adf39e1 to shade background, to give a visual separator effect, analogous to a normal set separator.

**Comment by eric-gilbertson** [comment]
I don't see any benefit in providing visual distinction to the NME entries. In fact I found the shading distracting when doing read backs on my show yesterday. My recommendation is to remove it. I don't know whether it is related to the shading but the up/down arrows did not seem to work for the NME entries.

**Comment by RocketMan** [comment]
The benefit is that it no longer a wall of text.  Just as set separators provide a break between sets, a log entry does, in fact, punctuate sets as well, so it makes sense to represent it visually as a kind of separator.

It has the added advantage that a DJ can scan quickly through his show by sets, versus having to read every line to find the start of sets. The PD can also quickly find LIDs, etc.

If the background is too dark, then we can lighten it or try some other effect.

Up/down arrows are orthogonal.  This is just a css style, analogous to songRow.  Which, by the way, we're missing songRow.  Is that intentional?

**Comment by RocketMan** [comment]
Visual cues are vexing.  I have removed all of them for now.

The idea is that in a spreadsheet, rows which do not follow the columnar layout need to be set off in some way.  Not highlighted, but set off visually, so it is clear they are not part of the matrix.  Often, it is done with indenting and italics.  I tried that first, but it looked like crap.  The other alternatives I tried did not look much better.  They tended either to highlight the row (not wanted) or clutter (definitely not wanted).

I am not too fused about this; it can go live without any visual cues, but ideally, there should be something to set these rows off.
